src/net/SocketUtil.hxx

```diff
@@ -32,6 +32,9 @@ class SocketAddress;
 /**
  * Creates a socket listening on the specified address.  This is a
  * shortcut for socket(), bind() and listen().
+ * When a unix socket is created (domain == AF_UNIX), its
+ * permissions will be striped down to prevent unauthorized
+ * access until proper permissions are applied at a later point.
```

Comment:
Should clarify that the caller is responsible for this "applying at a later point". Important parts of the contract should be clear enough so nobody can possibly misunderstand.

Reply:
Just removed that "manually" word before pushing because I thought it would be sufficient this ways :)
But valid point, give me a minute.